translate/rus.json

```diff
@@ -105,7 +105,7 @@
 "dark":"темный",
 "date":"дата",
 "DE":"СЛВ",
-"decisive openings":"решающие дебюты",
+"decisive openings":"результативные дебюты",
```

Comment:
I see your point in your translation @octopulo, but it doesn't tell me the full story. I would be like "yeah, some engine must have won the debut, but what counts in this category specifically?". With new translation it is a bit easier path to understand this (that a decisive opening is 1.5 - 0.5 score on it).